Configure.cmake

```diff
@@ -156,10 +165,21 @@ if(LD_SIZE GREATER "9")
   set(COMPILER_SUPPORTS_LONG_DOUBLE 1)
 endif()
 
+# Bug workaround for gcc 4.x
+if(CMAKE_C_COMPILER_ID MATCHES "GNU")
+  if (CMAKE_C_COMPILER_VERSION VERSION_LESS 5.0)
+    set(COMPILER_SUPPORTS_LONG_DOUBLE 0)
+  endif()
+endif()
+
 CHECK_C_SOURCE_COMPILES("
   int main() { __float128 r = 1;
   }" COMPILER_SUPPORTS_FLOAT128)
 
+if (${RUNNING_ON_TRAVIS} AND CMAKE_C_COMPILER_ID MATCHES "Clang")
```

Comment:
I really think we shouldn't have any Travis-aware variable in the build system. What is the exact error that forces you to add this condition?

Reply:
This is because linking with OpenMP fails on Travis although detection succeeds.